File: Day229_2025/Q229_GFG.cpp
class Solution {
  public:
    void rearrange(vector<int> &arr, int x) {
        // Q. Sort by Absolute Difference
        // code here
        stable_sort(arr.begin(), arr.end(), [x](int a, int b) {
            return abs(a - x) < abs(b - x);
        });
        return;
    }
};

File: Day229_2025/Q229_GFG.java
import java.util.*;

class Q229_GFG {
    public void rearrange(int[] arr, int x) {
        Arrays.sort(arr, new Comparator<Integer>() {
            public int compare(Integer a, Integer b) {
                return Math.abs(a - x) - Math.abs(b - x);
            }
        });
    }
}
